model/relation/repository/rdf/RdfMediaRelationRepository.php

```diff
@@ -227,6 +230,16 @@ private function findAllMediaByTarget(string $targetId, string $type): MediaRela
         return $this->mapSourceRelations($type, (array)$result, $targetId);
     }
 
+    public function getItemAssetUris(string $itemUri): array
+    {
+        $statement = $this->getPersistence()->query(
+            'SELECT subject FROM statements WHERE predicate = ? AND object = ?',
```

Comment:
So we make sure there are no duplications:

```suggestion
            'SELECT DISTINCT subject FROM statements WHERE predicate = ? AND object = ?',
```

Reply:
changed: b25eeca